src/client/game/symbols.hpp

```diff
@@ -131,6 +131,9 @@ namespace game
 	WEAK symbol<void(client_s* cl_0, svscmd_type type, const char* fmt, ...)> SV_SendServerCommand{0x0, 0x140537F10};
 	WEAK symbol<bool(int clientNum)> SV_IsTestClient{0x14224B5C0, 0x14052FF40};
 
+	// Utils
+	WEAK symbol<const char* (const char* str)> I_CleanStr{ 0x1422E9C10, 0x140580E80 };
```

Comment:
remove the spaces inside the brackets and it's perfect :P
```suggestion
	WEAK symbol<const char* (const char* str)> I_CleanStr{0x1422E9C10, 0x140580E80};
```

Reply:
question is, does this take a const char* or a char* ? i think it modifies the buffer that comes in, so maybe changing it to that would be good: 
```suggestion
	WEAK symbol<const char* (char* str)> I_CleanStr{0x1422E9C10, 0x140580E80};
```